### ISSUE TITLE
20190422 - Dat Night

### ISSUE THREAD

**Issue by whoisgina** [dat-land/comm-comm#47]
Offline meeting in Portland, OR, United States at Rumors

Time: 22 Apr 2019, 18:00 America/Los_Angeles

https://datnight.org

**Comment by dpaez** [comment]
Hi @whoisgina 👋 Hey if you have any notes or maybe some material/links/slides anything that comes from the last datnight it would be nice to link those with this issue. 

Thanks!

**Comment by whoisgina** [comment]
Hi @dpaez! This was mostly a work night/study hall, so I have no slides to share! I gave a brief intro to Dat & Beaker Browser and why we should be exploring this way of working and building, and then we split into groups.
- I shared these zines: `dat://classic-notebook-style.solarpunk.cool/` and `dat://home-page.angblev.com/`; some attendees made editable copies of them in Beaker Browser and started making their own. 
- A lot of the attendees brainstormed what an online/offline Dat zine could be about/look like with @calebmisclevitz; there are photos of that below!
- I set up Dat on a Raspberry Pi in the back of the room (following [this guide](https://github.com/new-computers/guides/blob/master/guides/installing-dat-raspberry-pi.md), thanks @new-computers), and had some interesting conversations with people about what we could use it for (always on peer! streaming video of future events! resource hub for the community! a concert?!)
- Lots of people eating snacks, chatting, working on side projects, and helping each other out!

The next one will be in May after Data Terra Nemo! I'll open a new issue once the date is set. 🌟 

![image_from_ios_1024](https://user-images.githubusercontent.com/2815123/56832276-57099a00-6820-11e9-8af4-8af02da436d9.jpg)
![D44H5AAUUAAH1Zz](https://user-images.githubusercontent.com/2815123/56832294-638df280-6820-11e9-86dd-df8950b3fceb.jpg)
![image_from_ios](https://user-images.githubusercontent.com/2815123/56832297-67217980-6820-11e9-9e57-34fdbc81b25c.jpg)
